packages/workspace/src/browser/workspace-service.ts

```diff
@@ -208,3 +422,19 @@ export interface WorkspaceInput {
     preserveWindow?: boolean;
 
 }
+
+export type WorkspaceRootNode = FileStatNode & ExpandableTreeNode;
+export namespace WorkspaceRootNode {
```

Comment:
It should belong to the navigator tree. `WorkspaceService` should not have UI dependencies